src/portal/item.rs

```diff
@@ -88,6 +88,18 @@ impl Item {
         self.secret = secret.as_ref().to_vec();
     }
 
+    /// The UNIX time when the item was created.
+    pub fn created(&self) -> Duration {
```

Comment:
Can't we have a Duration directly in the struct?

Reply:
Those are serialized/deserialized, no idea tbh

Reply:
Right, Duration is serialized as a `{"secs": 10, "nanos": 20}` but implementing a custom Serialize/Deserialize for Item should be pretty easy

Reply:
Let us land this as is for now